.travis.yml

```diff
@@ -1,9 +1,27 @@
+sudo: False
+
 language: python
-before_install:
-  - sudo apt-get -qq update
-  - sudo apt-get install -y libsnappy-dev
-  - pip install python-snappy
+
+matrix:
+  include:
+    - python: 3.5
+
 install:
-  - pip install tox
+  # Install conda
+  - wget http://repo.continuum.io/miniconda/Miniconda-latest-Linux-x86_64.sh -O miniconda.sh
```

Comment:
FYI I've had problems with http:// and conda recently.  Recommend to switching to https://